go.mod

```diff
@@ -193,6 +190,7 @@ require (
 	github.com/tklauser/go-sysconf v0.3.12 // indirect
 	github.com/tklauser/numcpus v0.6.1 // indirect
 	github.com/tredoe/osutil v1.5.0 // indirect
+	github.com/twpayne/go-vfs/v4 v4.3.0 // indirect
```

Comment:
shouldn't this be v5? That's what you bumped to in the code.

Reply:
indirect, so it measn something in the deps is bringing it, but IIRC the indirect deps are not bumped by renovate directly